flow/scripts/plot_cells.py

```diff
@@ -0,0 +1,67 @@
+import json
+import pandas as pd
+import matplotlib.pyplot as plt
+import synth_keep
+
+# Load the first JSON file
+with open('objects/nangate45/bp_multi/base/synth_full.json', 'r') as file:
+    data_a = json.load(file)
+
+# Load the second JSON file
+with open('objects/nangate45/bp_multi/base/synth.json', 'r') as file:
+    data_b = json.load(file)
+
+# Get the cell counts for each module in each file
+# create dictionary of module names and their area, strip the \ in front of module name
+# Area of module \bsg_mem_1r1w_synth_width_p99_els_p2_read_write_same_addr_p0_harden_p0 is 1821.834000
+with open('reports/nangate45/bp_multi/base/synth_cmos.txt', 'r') as file:
+    cmos = json.load(file)
+    transistors = {name.replace('\\', ''):
+                   int(module['estimated_num_transistors'].replace('+', ''))
```

Comment:
Note that the '+' means there were gates that couldn't be counted and were assigned zero.